library/common/extensions/cert_validator/platform_bridge/c_types.h

```diff
@@ -27,7 +27,7 @@ typedef envoy_cert_validation_result (*envoy_validate_cert_f)(const envoy_data*
 /**
  * Function signature for calling into platform APIs to clean up after validation completion.
```

Comment:
long as you're in here, validation is complete?

Reply:
Done!